types/rusha/index.d.ts

```diff
@@ -2,34 +2,35 @@
 // Project: https://github.com/srijs/rusha#readme
 // Definitions by: Jacopo Scazzosi <https://github.com/jacoscaz>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.0
 
 /// <reference types="node" />
 
-interface Hash {
-    update(value: string|number[]|ArrayBuffer|Buffer): Hash;
+interface RushaHash {
```

Comment:
this is a Hash, why renaming existing symbols? (think maintaince)
https://github.com/srijs/rusha/blob/master/src/hash.js#L6

Reply:
What is the best practice here? I had to rename `Worker` into `RushaRowker` as, with the addition of the `DOM` library, it had started shadowing `Worker`. I thought renaming `Hash` into something more specific would make things more uniform.

Reply:
as it is late here, I would say, we're describing existing bestiary, so we keep existing names. When we produce brand new beast, we give it new name. Applies to types and objects as well as to beasts. 
Just think about users of your your definition, they would not expect `RushaHash`, it does not existisn in the package readme. THey would expect 'Hash'

Reply:
Done, renamed `RushaHash` back to `Hash`